app/views/admin/users/index.html.erb

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>All da Users</title>
+</head>
+<body>
+    <table>
```

Comment:
The index view of users will be rendered into the `yield` of our application.html.erb so we do not want to include the full HTML and header just the content to be displayed in the body.

Reply:
just fixed!